common-system.js

```diff
@@ -239,7 +239,7 @@ System.prototype.loadSystemDescription = function loadSystemDescription(location
             throw error;
         }
     }, function (error) {
-        error.message = "Can't load package " + JSON.stringify(name) + " at " +
+        error.message = "Can't load package " + JSON.stringify(location) + " at " +
```

Comment:
After this fix on master we then have:
```
~/system v1.0.6-1-gafd0107 % node bundle.js boot-entry.js
/Users/joshua/system/node_modules/q/q.js:876
    throw error;
    ^

Error: Can't load package "file:///Users/joshua/system/node_modules/q/node_modules/weak-map/" at "file:///Users/joshua/system/node_modules/q/node_modules/weak-map/" because ENOENT: no such file or directory, open '/Users/joshua/system/node_modules/q/node_modules/weak-map/package.json'
    at Error (native)
```

So it's failing on an ENOENT similar to the 404 I set out to fix for the browser; this PR makes the ENOENT go away, but the bundle never completes...